src/GlobalStyles.js

```diff
@@ -65,9 +65,18 @@ main {
         border-radius: 8px;
         border: solid 1px var(--secondary-40);
         padding: 12px;
+
+        :focus {
+            border: solid 2px var(--primary-light);
+            outline: none;
+        }
+
+        :required:invalid {
+             box-shadow: none;
+        }
     
         ::placeholder {
-          color: var(--primary-light);
+            color: var(--primary-light);
```

Comment:
not sure whether this is just a display error, but there seems to be a sort of whitespace or checkbox before `color` here?